selfdrive/car/toyota/interface.py

```diff
@@ -110,11 +121,14 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), vin="", has_relay
       stop_and_go = False
       ret.safetyParam = 100
       ret.wheelbase = 2.70
-      ret.steerRatio = 18.27
+      ret.steerRatio = 17.8
```

Comment:
The value here indicated is just the starting point for the online-learner.

The value should come from whatever the parameter learner converges to. The nominal steer-ratio, for different reasons, might not be correctly representing the steer ratio relevant for vehicle modeling.

Reply:
Odd then. My learned steer ratio in LiveParameters is `18.720829857509521`, however that's usually too much and around curves, it oversteers and goes into a loop of lessening up on the steering too much after it oversteers, then oversteering, then lessening up on the steering.

Reply:
This shows what I mean (from 0.6.3 on stock openpilot): https://streamable.com/3773o

Reply:
Weird then, my learned steer ratio from LiveParameters is `18.72` however that's usually too high for curves as it oversteers, then goes into a loop of understeering to compensate. I read from an article on how to tune steering that reducing steerKf can have a similar effect of reducing oversteering rather than changing the steer ratio. Is that correct? Perhaps I should tune that and revert the steer ratio back to the learned value.

Reply:
This shows what I mean (from 0.6.3 on stock openpilot): https://streamable.com/3773o
Part of it is from my car's left hugging issue